interface/drugs/add_edit_drug.php

```diff
@@ -372,263 +368,235 @@ function sel_related() {
+                        "pw.pw_warehouse = lo.option_id WHERE " .
+                        "lo.list_id = 'warehouse' AND lo.activity = 1 ORDER BY lo.seq, lo.title",
+                        array($drug_id)
+                    );
+                    while ($pwrow = sqlFetchArray($pwres)) {
+                        $pwarr[] = $pwrow;
+                        echo "     <td class='align-top text-nowrap'>" . text($pwrow['title']) . "</td>\n";
+                    }
+                    ?>
+            </tr>
+            <tr>
+                <td class="align-top text-nowrap"><?php echo xlt('Min'); ?>&nbsp;</td>
+                <td class="align-top">
+                    <input class="form-control" size='5' name='form_reorder_point' maxlength='7' value='<?php echo attr($row['reorder_point']) ?>' title='<?php echo xla('Reorder point, 0 if not applicable'); ?>' data-toggle="tooltip" data-placement="top" />
+                </td>
+                <?php
+                foreach ($pwarr as $pwrow) {
+                    echo "     <td class='align-top'>";
+                    echo "<input class='form-control' name='form_wh_min[" .
+                    attr($pwrow['option_id']) .
+                    "]' value='" . attr(0 + $pwrow['pw_min_level']) . "' size='5' " .
+                    "title='" . xla('Warehouse minimum, 0 if not applicable') . "' data-toggle='tooltip' data-placement='top' />";
+                    echo "&nbsp;&nbsp;</td>\n";
+                }
+                ?>
+            </tr>
+            <tr>
+                <td class="align-top text-nowrap"><?php echo xlt('Max'); ?>&nbsp;</td>
+                <td>
+                    <input class='form-control' size='5' name='form_max_level' maxlength='7' value='<?php echo attr($row['max_level']) ?>' title='<?php echo xla('Maximum reasonable inventory, 0 if not applicable'); ?>' data-toggle="tooltip" data-placement="top" />
+                </td>
+                <?php
+                foreach ($pwarr as $pwrow) {
+                    echo "     <td class='align-top'>";
+                    echo "<input class='form-control' name='form_wh_max[" .
+                    attr($pwrow['option_id']) .
+                    "]' value='" . attr(0 + $pwrow['pw_max_level']) . "' size='5' " .
+                    "title='" . xla('Warehouse maximum, 0 if not applicable') . "' data-toggle='tooltip' data-placement='top' />";
+                    echo "</td>\n";
+                }
+                ?>
+            </tr>
+        </table>
+    </div>
+
+    <div class="form-group mt-3 drugsonly">
+        <label class="font-weight-bold"><?php echo xlt('Pill Size'); ?>:</label>
+        <?php
+            generate_form_field(array('data_type' => 1,'field_id' => 'form','list_id' => 'drug_form','empty_title' => 'SKIP'), $row['form']);
+        ?>
+    </div>
+
+    <div class="form-group mt-3 drugsonly">
+        <label class="font-weight-bold"><?php echo xlt('Form'); ?>:</label>
```

Comment:
It looks like the `Form` and `Pill Size` labels were swapped here?

Reply:
Oops!! I will fix it

Reply:
@bradymiller  Update the coomit :)